.eslintrc.json

```diff
@@ -16,7 +16,8 @@
   "globals": {
     "__SERVER__": true,
     "__CLIENT__": true,
-    "__PRODUCTION__": true
+    "__PRODUCTION__": true,
+    "jasmine": true
```

Comment:
It's used by the e2e tests to increase the default timeout interval of Jasmine.

```javascript
  /**
   * Increase the default Jasmine timeout interval because our tests are taking
   * longer.
   */
  beforeEach(() => {
    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
  });
```